src/GlobalScreen/Scope/Notification/Factory/AbstractBaseNotification.php

```diff
@@ -41,8 +57,47 @@ public function getProviderIdentification() : IdentificationInterface
     /**
      * @inheritDoc
      */
-    public function getRenderer() : NotificationRenderer
+    public function getRenderer(UIFactory $factory) : NotificationRenderer
```

Comment:
I like that! If there is some time I will introduce a RendererInterface für all Scopes which should use this